03-cartography/README.md

```diff
@@ -1,43 +1,78 @@
 # Cartography with CARTO
 
-## Agenda
+1. [Cartographic Design](#design)
+   - [Talks](#talks)
+   - [CartoCSS](cartocss)
+   - [TurboCARTO](#turbo)
+2. [Exercises](#design)
+3. [Resources](#design)
+   - [Colors](#design)
+   - [Other resources](#resources)
```

Comment:
I'd remove an index that points to just a list of links.

Reply:
Removed.